packages/ui/src/proposals/helpers/getDetailsRenderStructure.ts

```diff
@@ -201,6 +202,16 @@ const booleanMapper: Mapper<UpdateChannelPayoutsDetail, 'channelCashoutsEnabled'
   ]
 }
 
+const palletStatusMapper: Mapper<UpdatePalletFrozenStatusDetail, 'frozen'> = (value) => {
+  return [
+    {
+      label: 'Proposed Status',
+      renderType: 'Text',
+      value: value ? 'Enable' : 'Disable',
```

Comment:
I think it's the opposite if we want to freeze the pallet the proposed status is "Disable" otherwise "Enabled".

Also please rename value to freezePallet